src/local/butler/lint.py

```diff
@@ -15,10 +15,140 @@
 
 import os
 import sys
+import yaml
 
 from local.butler import appengine
 from local.butler import common
 
+_GOLINT_EXCEPTIONS = [
+    'types.go'  # Not all model names conform to Go naming conventions.
+]
+_LICENSE_CHECK_FILENAMES = ['Dockerfile']
+_LICENSE_CHECK_EXTENSIONS = [
+    '.bash',
+    '.c',
+    '.cc',
+    '.cpp',
+    '.css',
+    '.h',
+    '.htm',
+    '.html',
+    '.js',
+    '.go',
+    '.py',
+    '.sh',
+    '.yaml',
+]
+_LICENSE_CHECK_IGNORE_FILENAMES = ['technology.css']
+_LICENSE_CHECK_IGNORE_DIRECTORIES = [
+    'third_party',
+    'templates',  # Generated code.
+]
+_LICENSE_CHECK_STRING = 'http://www.apache.org/licenses/LICENSE-2.0'
+_PY_TEST_SUFFIX = '_test.py'
+_PY_INIT_FILENAME = '__init__.py'
```

Comment:
Globals are sometimes better since 
1. easier to modify - e.g. _LICENSE_CHECK_EXTENSIONS
2. if you have do local, you have to do it lowercase (otherwise lint complains). lowercase does not make look like constant.
_PY_INIT_FILENAME was used twice, so just thought to keep it in constant.